Firestore/core/src/firebase/firestore/remote/grpc_connection.cc

```diff
@@ -50,6 +51,8 @@ namespace {
 
 const char* const kXGoogAPIClientHeader = "x-goog-api-client";
 const char* const kGoogleCloudResourcePrefix = "google-cloud-resource-prefix";
+const char* const kFirebaseFirestoreHeartbeatKey = "X-firebase-client-log-type";
```

Comment:
These keys should be hidden in the metadata provider specific to the firebase client, as we did on Android.